doc/bash/git_aliases.md

```diff
@@ -22,6 +22,8 @@
 
 ### Git branch ###
 - **gb**: `git branch -v`
+- **gba**: `git for-each-ref --sort=committerdate refs/heads/ --format="%(authordate:short) %(color:red) (objectname:short) %(color:yellow)%(refname:short)%(color:reset) (%(color:green)%(committerdate:relative)%(color:reset))"`
```

Comment:
Please add a short description of the command and remove the actual command. The command is too long and I think it doesn't make sense to be in the documentation file.

Reply:
By the way, what is the difference between `gba` and `gbd`? I wasn't able to spot any difference in the commands.

Reply:
The difference is at the sort order: 
`gba`: --sort=committerdate
`gbd`: --sort=-committerdate

The `-` sign says the sort will be descending on the specified column.